native/acter/src/api/news.rs

```diff
@@ -431,78 +431,159 @@ impl NewsEntryDraft {
         Ok(true)
     }
 
-    pub fn add_audio_slide(
+    pub async fn add_audio_slide(
```

Comment:
same here ... these do not seem to be related to the sdk-upgrade, are they?

Reply:
Fixed